src/xdocs/cmdline.xml.vm

```diff
@@ -147,7 +147,7 @@ java -D&lt;property&gt;=&lt;value&gt;  \
         Note that the <code>-n packageNamesFile</code>
         option has been dropped for Checkstyle 5.0, because of
         significant changes regarding package name file handling. See <a
-        href="config.html#Packages"/> for details.
+              href="config.html#Packages"> for details.</a>
```

Comment:
Put whole tag on same line.

Fix commit message to have no space before ':'

Reply:
[ERROR] [checkstyle] [ERROR] E:\CheckStyle\checkstyle\checkstyle\src\xdocs\cmdline.xml.vm:149: Line should not be longer than 100 symbols [lineLengthXml]
Putting the whole tag on same line gives this error.

Reply:
Ok. 
![image](https://user-images.githubusercontent.com/29230917/201537404-a3f0cca2-70f6-4c18-a00a-b1c060afd1c8.png)
I guess, now that error should not come.

Reply:
Done.